packages/common/src/hooks/useHostContext.ts

```diff
@@ -3,13 +3,18 @@ import create from 'zustand';
 import { LocalStorage } from '../localStorage';
 
 import { SupportedLocales } from '../intl/intlHelpers';
+import { Store, User } from '../types';
 
 type HostContext = {
   setAppBarContentRef: (ref: React.MutableRefObject<null> | null) => void;
   appBarButtonsRef: React.MutableRefObject<null> | null;
   appBarContentRef: React.MutableRefObject<null> | null;
   locale: SupportedLocales;
   setLocale: (locale: SupportedLocales) => void;
+  setStore: (store: Store) => void;
```

Comment:
probably not the best place for it - just wanted to source the values from somewhere stateful